src/main/scala/org/dberg/hubot/listeners/TestListener3.scala

```diff
@@ -0,0 +1,21 @@
+package org.dberg.hubot.listeners
+
+import org.dberg.hubot.Hubot
+import org.dberg.hubot.listeners.Listener.CallbackSuccess
+import org.dberg.hubot.models._
+
+class TestListener3(hubot: Hubot) extends Listener(hubot) {
+
+  val callback: Callback = {
```

Comment:
Unless we have reason for a listener to return a meaningful value, Try[Unit] seems like a fine option, except the user can of course still do `val result = somethingThatThrows(); Success(result)`, so the call would have to be wrapped anyway. So to me, the only difference is whether the user has to think about error handling when implementing a listener.